### ISSUE TITLE
throw NullPointerException if DialogFragment implements onCreateDialog

### ISSUE THREAD

**Issue by edwardaa** [Manabu-GT/EtsyBlur#4]
implement onCreateDialog() instead of onCreateView(), an exception raised:

java.lang.NullPointerException
            at com.ms.square.android.etsyblur.Util.animateAlpha(Util.java:61)
            ...
            at com.example.AlertDialogFragment.onStart(AlertDialogFragment.java:57)
            at android.support.v4.app.Fragment.performStart(Fragment.java:1810)
            ...

**Comment by Manabu-GT** [comment]
This issue has been fixed in v0.1.1. 

The below is an example of using an alert dialog.
If you override onCreateDialog(), do not return non null view from onCreateView().

```java
@Override
public Dialog onCreateDialog(Bundle savedInstanceState) {
    AlertDialog dialog = new AlertDialog.Builder(getActivity(), getTheme()) .setIcon(R.drawable.ic_launcher)
.setTitle("Hello")
.setPositiveButton("OK", null)
.create();
dialog.getWindow().getAttributes().height = WindowManager.LayoutParams.WRAP_CONTENT;
return dialog;
}
```

**Comment by edwardaa** [comment]
Thanks :)
